crowbar_framework/app/models/service_object.rb

```diff
@@ -219,8 +219,7 @@ def add_pending_elements(bc, inst, elements, queue_me, pre_cached_nodes = {})
           next if node.nil?
           pre_cached_nodes[node_name] = node
 
-          # Make sure the node is allocated
-          node.allocate!
+          # Mark node as pending. User will be informed about manual allocation
```

Comment:
Can you add a "if node is not allocated" at the end of this comment? Otherwise, it's a bit confusing.